js/bitmart.js

```diff
@@ -2771,6 +2772,94 @@ module.exports = class bitmart extends Exchange {
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+        };
+        if (limit !== undefined) {
+            request['N'] = limit;
+        }
+        if (since !== undefined) {
+            request['start_time'] = since;
+        }
+        const response = await this.privateGetSpotV1MarginIsolatedBorrowRecord (this.extend (request, params));
+        //
+        //     {
+        //         "message": "OK",
+        //         "code": 1000,
+        //         "trace": "9bc89822-aa8c-4a43-a52f-3a9a2719bad4",
+        //         "data": {
+        //             "records": [
+        //                 {
+        //                     "borrow_id": "1657664327844Lk5eJJugXmdHHZoe",
+        //                     "symbol": "BTC_USDT",
+        //                     "currency": "USDT",
+        //                     "borrow_amount": "20.00000000",
+        //                     "borrow_unpaid": "0.00000000",
+        //                     "daily_interest": "0.00055000",
+        //                     "hourly_interest": "0.00002291",
+        //                     "interest_amount": "0.00045833",
+        //                     "interest_unpaid": "0.00000000",
+        //                     "create_time": 1657664329000
+        //                 },
+        //             ]
+        //         }
+        //     }
+        //
+        const data = this.safeValue (response, 'data', {});
+        const rows = this.safeValue (data, 'records', []);
+        const interest = this.parseBorrowInterests (rows, market);
+        return this.filterByCurrencySinceLimit (interest, code, since, limit);
+    }
+
+    parseBorrowInterest (info, market = undefined) {
+        //
+        //     {
+        //         "borrow_id": "1657664327844Lk5eJJugXmdHHZoe",
+        //         "symbol": "BTC_USDT",
+        //         "currency": "USDT",
+        //         "borrow_amount": "20.00000000",
+        //         "borrow_unpaid": "0.00000000",
+        //         "daily_interest": "0.00055000",
+        //         "hourly_interest": "0.00002291",
+        //         "interest_amount": "0.00045833",
+        //         "interest_unpaid": "0.00000000",
+        //         "create_time": 1657664329000
+        //     },
+        //
+        const marketId = this.safeString (info, 'symbol');
+        market = this.safeMarket (marketId, market);
+        const timestamp = this.safeInteger (info, 'create_time');
+        return {
+            'symbol': this.safeString (market, 'symbol'),
+            'marginMode': 'isolated',
+            'currency': this.safeCurrencyCode (this.safeString (info, 'currency')),
+            'interest': this.safeNumber (info, 'interest_unpaid'),
+            'interestRate': this.safeNumber (info, 'hourly_interest'),
+            'amountBorrowed': this.safeNumber (info, 'borrow_unpaid'),
```

Comment:
Question, shouldn't the `amountBorrowed` be `borrow_amount` instead of `borrow_unpaid`? 
![image](https://user-images.githubusercontent.com/43336371/181047237-55746ea0-e56c-4f6d-9eb8-d552e7edfef9.png)
![image](https://user-images.githubusercontent.com/43336371/181047310-1e403ebd-ef15-488e-8cc6-2812fbd081e2.png)

Reply:
I used 'borrow_unpaid' and 'interest_unpaid' because I think this method is generally used to show the amount that needs to be repaid and if I use 'borrow_amount' it will show the entire borrow amount history and how much the user needs to repay isn't as clear.

Reply:
@Dan-krm I see but tbh I'm a bit confused because I don't even see that field in the raw responses/ documentation. Do you know why?

![image](https://user-images.githubusercontent.com/43336371/181197730-7d4f6d7e-6642-4aa2-96be-b14e8f977a5f.png)
![image](https://user-images.githubusercontent.com/43336371/181197748-ceac645f-ae9a-43c6-a897-c7b2599597cf.png)

Reply:
I'm not sure why that field isn't shown in your response, maybe it isn't available on all trading pairs yet, I changed 'borrow_unpaid' to 'borrow_amount' to be safe, I think 'interest_unpaid' is more important to leave as is but your response isn't showing that either.

Reply:
@Dan-krm yeah very weird, maybe the API changed in the meantime? Cause they're also not in the docs  https://developer-pro.bitmart.com/en/spot/#get-borrow-record-isolated

Reply:
Yeah I did some more testing and it looks like 'borrow_unpaid' and 'interest_unpaid' were both removed